lib/upgrader.js

```diff
@@ -73,46 +86,42 @@ function replace({ argv }) {
     [ 'apos.areas.plaintext', 'apos.area.plaintext' ],
     [ 'apos.areas.walk', 'apos.area.walk' ]
   ];
-  for (const file of files) {
-    const original = fs.readFileSync(file, { encoding: 'utf8' });
-    let transformed = original;
-    for (const [ pattern, replacement ] of replaces) {
-      transformed = replaceAll(transformed, pattern, replacement);
-    }
-    if (original !== transformed) {
-      fs.writeFileSync(file, transformed);
-    }
-  }
+
+  replaceOccurences(files, replaces);
+}
+
+function renameModulesDeclarations(isMultisite) {
+  const files = [
+    'app.js',
+    ...isMultisite
+      ? [
+        'dashboard/index.js',
+        'sites/index.js',
+        ...globByPattern('dashboard/lib/theme-*.js'),
+        ...globByPattern('sites/lib/theme-*.js')
+      ]
+      : []
+  ];
+
+  const renamedReplaces = Object.entries(renamedModulesMap)
+    .map(([ old, newName ]) => {
+      const oldName = old.replace('dashboard/', '').replace('sites/', '');
+
+      return [ oldName, newName ];
+    }, []).filter(([ oldName, newName ]) => !legacyModuleNameMap[oldName]);
+
+  const replaces = [ ...renamedReplaces, ...Object.entries(legacyModuleNameMap) ];
```

Comment:
Maybe we could have only used `legacyModuleNameMap` here, but I notcie that the tool also renames modules if they have a name `property`.